### PR TITLE
Added missing jaeger flag to Thanos store.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: environments/openshift/manifests/observatorium-template.yaml [hunk 1]
```diff
@@ -1190,7 +1197,7 @@ parameters:
 - name: NAMESPACE
   value: telemeter
 - name: THANOS_IMAGE
-  value: improbable/thanos
+  value: quay.io/quay/thanos
```
Comment (kakkoyun) [inline environments/openshift/manifests/observatorium-template.yaml]: Just for your information, we override these variables before we deploy them. That's why they are a bit outdated.

**Comment by kakkoyun** [comment]
@bwplotka There are issues with the formatting I believe.

**Comment by bwplotka** [comment]
PTAL all should be good now (:

File: environments/openshift/manifests/observatorium-template.yaml (hunk 1 above)
Comment (metalmatze) [reply environments/openshift/manifests/observatorium-template.yaml]: And it should be `quay.io/thanos-io/thanos`.

**Pull request merged**